src/shell/powershell.rs

```diff
@@ -48,4 +49,8 @@ impl Shell for PowerShell {
     fn to_clap_shell(&self) -> clap_complete::Shell {
         clap_complete::Shell::PowerShell
     }
+
+    fn format_path(&self, path: &str) -> Result<String> {
+        Ok(path.to_string())
```

Comment:
powershell can be run on Mac and Linux (not sure about usage though). Maybe we need to call `unixpath` here too? We probably can't know if we're in WSL or Cygwin, or do we? based on the infer_shell infra and the process-walking method?
What I'm thinking is--maybe it's not shell-related, but environment related, and therefore if we have this information?

We can start without any inference, so it will require the following, if one wants to use the Windows version of fnm within Cygwin or WSL:

```
fnm env --path-formatter=cygwin
fnm env --path-formatter=wsl
```

what do you think about it?

Reply:
Actually, I figured out a way to detect if the executable is being run from WSL, but inferring the shell is not possible and the user needs to provide it as a flag (maybe there can be a custom error message for it).
Detecting WSL works the same way as shell inferring, at some point up the parent process chain there is a process named "wsl.exe", but there's no information on the shell being used.